src/style.css

```diff
@@ -0,0 +1,80 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.nav {
+  display: flex;
+  height: 60px;
+}
+
+.logo {
+  display: flex;
```

Comment:
Why do you need here display: flex? I suppose, you want to stretch it to full height doing this, but you just may center your logo by using `align-self: center`, because it's situated in flex container
```suggestion
```